internal/config/default/config.toml

```diff
@@ -18,7 +18,7 @@ limit = 0
   quit = ["q"]
   help = ["?"]
   describe = ["D"]
-  edit = ["e"]
+  edit = ["e", "alt+e"]
```

Comment:
Similarly here, `alt+e` is not captured on my Mac and I would have to change this to something else, then because of the `Alt` check, it wouldn't work.